ffi/.golangci.yaml

```diff
@@ -36,7 +36,6 @@ linters:
     - bodyclose
     - copyloopvar
     - depguard
-    - dupword
```

Comment:
discussion around this change is here: https://github.com/ava-labs/avalanchego/pull/4146/files#r2248670142